src/resources/UI/check-symbol.svg

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```

Comment:
Then the question is, can we install it? No worries for now, can you make one task to introduce *font-awesome* and replace relevant SVG files with it? Thanks!